### ISSUE TITLE
Media info doesn't always load when checking multiple files

### ISSUE THREAD

**Issue by Ryrynz** [Garteal/MPDN_Extensions#21]
If there's significant lag (Not responding message displays) when determining the media info the subsequent media info results are not loaded.

If I select six new files from the playlist (currently residing all on a networked HDD, thus access is somewhat slow) and check the media info only a few will load. If done a second time all will load.

**Comment by Garteal** [comment]
Yeah this is something that you'll have to deal with as the HDD probably has to spin up and the network delay (if not locally) will both affect the situation. I'm unsure whether I want to fix it in the extension or not.
